BundID/Managers/StorageManager.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+class StorageManager: StorageManagerType {
+    
+    private var userDefaults: UserDefaults
+    
+    var setupCompleted: Bool {
+        get {
+            userDefaults.bool(forKey: StorageKey.setupCompleted.rawValue)
+        }
+        set {
+            userDefaults.set(newValue, forKey: StorageKey.setupCompleted.rawValue)
+        }
+    }
+    
+    init(userDefaults: UserDefaults = .standard) {
+        self.userDefaults = userDefaults
+        
+        if CommandLine.arguments.contains(LaunchArgument.setupCompleted) {
+            self.userDefaults.set(true, forKey: StorageKey.setupCompleted.rawValue)
+        }
```

Comment:
I would not expect that the init of this class would perform this side-effect. Maybe move the if into `BundIdApp`, just after the other launch argument check?

Reply:
:+1: